File: sapi/src/main/java/org/roylance/yaas/services/IUserDeviceService.kt
package org.roylance.yaas.services

interface IUserDeviceService {
	fun save(request: org.roylance.yaas.YaasModel.UIRequest): org.roylance.yaas.YaasModel.UIResponse
	fun all(request: org.roylance.yaas.YaasModel.UIRequest): org.roylance.yaas.YaasModel.UIResponse
}

File: api/javascript/IUserDeviceService.ts
/// <reference path="./YaasModel" />
export interface IUserDeviceService {
	save(request: org.roylance.yaas.UIRequest, onSuccess:(response: org.roylance.yaas.UIResponse)=>void, onError:(response:any)=>void)
	all(request: org.roylance.yaas.UIRequest, onSuccess:(response: org.roylance.yaas.UIResponse)=>void, onError:(response:any)=>void)
}